command/pr_create.go

```diff
@@ -409,6 +409,18 @@ var prCreateCmd = &cobra.Command{
 	Use:   "create",
 	Short: "Create a pull request",
 	RunE:  prCreate,
+	Example: `$ gh pr create
+$ gh pr create --fill
+$ gh pr create -t "PR With Title And Body" -b "This is the pull request body"
+$ gh pr create -t OpenAWebBrowserToCreateThisPR --web
+$ gh pr create -t DraftPR --draft
+$ gh pr create -t PRWithBaseBranch -b PRBody -B branchName
+$ gh pr create -t PRWithReviewers -b PRBody -r user1Login,user2Login
+$ gh pr create -t PRWithLabels -b PRBody -l label1,label2
+$ gh pr create -t PRWithProjects -b PRBody -p cli/1,cli/2
+$ gh pr create -t PRWithAssignees -b PRBody -a user1Login,user2Login
+$ gh pr create -t PRWithMilestone -b PRBody -m someMilestone
+`,
```

Comment:
Similar to the comment above, I made some tweaks here.

```suggestion
	Example: `$ gh pr create --title "The bug is fixed" --body "Everything works again"
$ gh pr create --reviewer user1Login,user2Login
$ gh pr create --label label1,label2
$ gh pr create --project cli/1,cli/2
$ gh pr create --author user1Login,user2Login`,
```

Reply:
Per https://github.com/cli/cli/pull/1167, let's also indent all code in examples with a single tab:
```go
Example: `
    $ foo
    $ bar
`,
```